vllm/v1/core/kv_cache_manager.py

```diff
@@ -154,6 +159,13 @@ def get_computed_blocks(
             # we shouldn't modify it directly.
             block_hashes.append(last_block_hash)
 
+        # Check the remote cache for the external prefix cache blocks.
+        if self.connector is not None:
+            computed_blocks =\
+                    self.connector.get_external_prefix_cache_blocks(
+                            request, computed_blocks,
+                            len(computed_blocks) * self.block_size, self)
```

Comment:
Passing the KV cache manager doesn't look like an ideal design. Can we somehow avoid it?